UPGRADE_NOTES.md

```diff
@@ -1,4 +1,55 @@
-### Upgrading from 0.14 to 0.15
+## Upgrading from 0.16 to 0.17
+
+- Ensure you've upgraded to crystal 0.30.0
+- Upgrade Lucky CLI (homebrew)
+
+```
+brew update
+brew upgrade crystal-lang # Make sure you're up-to-date. Requires 0.30.0
+brew upgrade lucky
+```
+
+- Upgrade Lucky CLI (Linux)
+
+- Update `.crystal-version` to `0.30.0`
+
+> Remove the existing Lucky binary and follow the Linux
+> instructions in this section
+> https://luckyframework.org/guides/getting-started/installing#on-linux
+
+- Update versions in `shard.yml`
+  - Lucky should be `~> 0.17`
+- Run `shards update`
+
+### General updates
+- Rename: Action rendering method `text` to `plain_text`.
+- Update: use of `number_to_currency` now returns a String instead of writing to the view directly.
+- Rename: `Avram::Repo` to `Avram::Database` in `config/database.cr`.
```

Comment:
This is slightly more involved. You need to create an `AppDatabase` class in `src/app_database.cr`. Then you need to change `Avram::Repo.configure` to `AppDatabase.configure`. You also need to do set the `database_to_migrate`. A few other changes, like requiring the app database are also required.

https://github.com/luckyframework/lucky_cli/commit/1fd4d7d961523711736df37dd205ff971e2fc0c2